locallib.php

```diff
@@ -197,3 +197,44 @@ function create_card($header, $text) {
     return $out;
 }
 
+/**
+ * Draws star gradings
+ *
+ * @param int $filled number of filled stars to draw
+ * @param int $total number of to draw in totaldraw_report
+ */
+function create_report_stars($filled, $total) {
+    $out = '';
+
+    for ($i = 1; $i <= $total; $i++) {
+        if ($i <= $filled) {
+            $out .= "\u{2605}";
+        } else {
+            $out .= "\u{2606}";
+        }
+    }
+
+    return $out;
+}
+
+/**
+ * Creates grading information container from report
+ *
+ * @param mixed $report object containing the report
+ */
+function create_report_grading($report) {
+    $out = html_writer::start_div('card row');
```

Comment:
```suggestion
    $out = html_writer::start_div('card row digitala-card');
```

And same addition to row 188 (add class *digitala-card*), I have forgotten to add margins for cards. If adding this, then this part needs to be in styles.css:
```
.digitala-card {
    margin-top: 1vh;
    margin-bottom: 1vh;
}
```